resources/js/script.js

```diff
@@ -0,0 +1,21 @@
+window.addEventListener('load', () => {
+    handleDelete()
+})
+
+function handleDelete() {
+    const elements = document.querySelectorAll('.deleteItemBtn')
+
+    elements.forEach(element => {
+        element.addEventListener('click', () => {
+            const url = element.dataset.url
```

Comment:
```suggestion
            const url = element.dataset.url;
```
thêm ; vào cuối dòng nha